jobs/haproxy/templates/haproxy_wrapper.erb

```diff
@@ -8,6 +8,8 @@ CONFIG=/var/vcap/jobs/haproxy/config/haproxy.config
 PID_FILE=/var/vcap/sys/run/haproxy/haproxy.pid
 DRAIN_LOCK=/var/vcap/sys/run/haproxy/drain.lock
 
+ENV_OVERRIDE=<$= p("ha_proxy.env_override", "") %>
```

Comment:
Would this work if you are providing multiple env vars? It would be safer to dump the env to a file and source it before launching HAProxy.

Reply:
That's fair. Ideally the env is only applicable to the launched HAProxy process. The intended use case was a `LD_PRELOAD` variable. Maybe I can make `haproxy.env` a list and process / concatenation properly.

Adding a subshell around this (the only option I see) that would limit the scope of the env variables would be a bit overkill I think.

Reply:
Just tried putting multiple vars into a single BOSH variable and that does not work.

Reply:
What about:
```
env $(grep -v '^#\|^$' ./env | tr '\n' ' ') haproxy ...
```
Possibly dropping the comment / empty line check as we can just forbid that.

Reply:
fair. `env $VARS -S` to make sure the params to haproxy are passed, but that could work. I will try these days.